### PR TITLE
Fix a Spanish legend text

### PR DESCRIPTION
**Pull request by bryant1410**
NB: It seems that GH changed other things by itself when editing from edit. I suspect it changed the line endings, but didn't check.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @recena @carlossg @andresrc @mafraba @raul-arabaolaza

File: core/src/main/resources/jenkins/model/Jenkins/legend_es.properties [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
+red=La última ejecución falló.
+red_anime=La última ejecución falló. Se ha lanzado una nueva ejecución.
 
-health-81plus=El estado de salud del proyecto supera el 80%. Puedes situar el rat�n sobre el icono para ver una explicaci�n detallada.
+health-81plus=El estado de salud del proyecto supera el 80%. Puedes situar el ratón sobre el icono para ver una explicación detallada.
 health-00to20=El estado de salud del proyecto es inferior al 20%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
 health-41to60=El estado de salud del proyecto est\u00E1 entre el 40% y el 60%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
 health-21to40=El estado de salud del proyecto est\u00E1 entre el 20% y el 40%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-61to80=El estado de salud del proyecto est\u00E1 entre el 80% y 80%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
+health-61to80=El estado de salud del proyecto est\u00E1 entre el 60% y 80%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
```
Comment (carlossg) [inline core/src/main/resources/jenkins/model/Jenkins/legend_es.properties]: This line LGTM. The rest seems to be line endings

**Comment by batmat** [comment]
@bryant1410 Thanks for your work! 
Could you please remove the whitespace modifications, and keep only the actually modified one? 

Our spanish-ready contributors seem to be OK with your change, so all is almost perfect :). We're just trying to avoid changes to unrelated lines in general to ease potential future diff-archeology.

Thanks!

**Comment by bryant1410** [comment]
Ok, only left this change.

**Comment by bryant1410** [comment]
Btw, the other changes were about encoding: GitHub changed the characters with accent marks.

**Pull request merged**